src/interface/storage.thrift

```diff
@@ -76,8 +91,8 @@ struct ResponseCommon {
 
 struct QueryResponse {
     1: required ResponseCommon result,
-    2: optional common.Schema vertex_schema,   // vertex related props
-    3: optional common.Schema edge_schema,     // edge related props
+    2: optional map<common.TagID, common.Schema>(cpp.template = "std::unordered_map")       vertex_schema,
```

Comment:
As we talked about offline,  combine all tags schema into one.

Reply:
Later I thought it would be better to do this (the last time I submitted that was the worst). 
The reasons are as follows:
1 In most cases there is only one tag id, then there is almost no difference between using only one schema at this time.
2 If there are multiple tag ids, the disadvantage of this approach is that it will output more schemas, but the advantage is that you don't need to fill in the extra data.
3 For the query engine, use map to save the schema, which is easier to process data.

Reply:
Well,  it's not a key problem.  Let's go on with your approach.